test/unit/utils.test.ts

```diff
@@ -939,7 +939,7 @@ describe('driver utils', function () {
     });
   });
 
-  describe('matchesParentDomain()', () => {
+  describe('checkParentDomainMatch()', () => {
```

Comment:
should we add analogous tests for what happens when we have a trailing dot on a 2- or 1- part srv?